frontend/src/__tests__/cypress/cypress/tests/mocked/modelRegistrySettings/modelRegistrySettings.cy.ts

```diff
@@ -596,6 +596,20 @@ describe('CreateModal', () => {
     modelRegistrySettings.resourceNameSelect.openAndSelectItem('sampleSecret', true);
     modelRegistrySettings.findExistingCAKeyInputToggle().should('be.enabled');
     modelRegistrySettings.keySelect.openAndSelectItem('bar.crt', true);
+    // Open the resource selector and verify both groups are initially visible
+    modelRegistrySettings.resourceNameSelect.findToggleButton().click();
+    cy.contains('ConfigMaps').should('be.visible');
+    cy.contains('Secrets').should('be.visible');
+
+    // Search for a value that exists in ConfigMaps but not in Secrets
+    modelRegistrySettings.resourceNameSelect
```

Comment:
I think here the comment should be `Search for a value that exists in Secrets but not in ConfigMaps`?